server/mscalendar/views/calendar.go

```diff
@@ -157,11 +152,22 @@ func RenderEventAsAttachment(event *remote.Event, timezone string, options ...Op
 			Value: event.Location.DisplayName,
 			Short: true,
 		})
+	}
+
+	if event.Conference != nil {
+		// Use conference URL as title link if theres conference data present
+		titleLink = event.Conference.URL
 
-		// Use location display name as link if can be parsed as an URL
-		if isKnownMeetingURL(event.Location.DisplayName) {
-			titleLink = event.Location.DisplayName
+		title := "Meeting URL"
+		if event.Conference.Application != "" {
+			title = event.Conference.Application
 		}
+
+		fields = append(fields, &model.SlackAttachmentField{
+			Title: title,
+			Value: event.Conference.URL,
```

Comment:
0/5, may be error-prone to do it my suggested way

```suggestion
			Value: titleLink,
```